desc/equilibrium/equilibrium.py

```diff
@@ -1243,18 +1248,80 @@ def compute_theta_coords(
 
         Returns
         -------
-        coords : ndarray, shape(k,3)
-            coordinates [rho,theta,zeta].
+        coords : ndarray
+            Shape (k, 3).
+            DESC computational coordinates [ρ, θ, ζ].
         info : tuple
-            2 element tuple containing residuals and number of iterations
-            for each point. Only returned if ``full_output`` is True
+            2 element tuple containing residuals and number of iterations for each
+            point. Only returned if ``full_output`` is True.
+
         """
         return compute_theta_coords(
             self,
             flux_coords,
             L_lmn=L_lmn,
+            tol=tol,
             maxiter=maxiter,
+            full_output=full_output,
+            **kwargs,
+        )
+
+    def map_clebsch_coords(
```

Comment:
Can this also be `compute_clebsch_coords` to align with `compute_theta_coords`?

Reply:
Counter: this is closer to `map_coordinates` name and

the theta in `compute_theta_coords` is DESC theta. In both functions we compute DESC theta, so `compute_clebsch_coords` is a bit weird

Reply:
I'm never going to use `eq.map_clebsch`, so I can remove that. I would like to use `map_clebsch`. We can redirecto to that through `map_coordinates`